apps/etterlatte-trygdetid/src/test/kotlin/no/nav/etterlatte/trygdetid/TrygdetidServiceImplIntegrationTest.kt

```diff
@@ -162,28 +163,45 @@ internal class TrygdetidServiceImplIntegrationTest {
                 sak = standardOpplysningsgrunnlag.sak,
                 metadata = standardOpplysningsgrunnlag.metadata,
             )
-
         repository.opprettTrygdetid(
             trygdetid(
                 behandlingId = behandlingId,
                 sakId = SecureRandom().nextLong(100_000),
+                ident = UKJENT_AVDOED,
                 opplysninger = emptyList(),
             ),
         )
 
         val trygdetid = runBlocking { trygdetidService.hentTrygdetid(behandlingId, saksbehandler) }
 
-        with(trygdetid?.opplysningerDifferanse!!) {
-            differanse shouldBe false
-            with(oppdaterteGrunnlagsopplysninger) {
-                avdoedFoedselsdato shouldBe null
-                avdoedDoedsdato shouldBe null
-                avdoedFylteSeksten shouldBe null
-                avdoedFyllerSeksti shouldBe null
-            }
+        with(trygdetid!!) {
+            trygdetid.ident shouldBe UKJENT_AVDOED
+            opplysningerDifferanse shouldBe null
         }
     }
 
+    @Test
+    fun `skal slette trygdetid hvis ident ikke lenger finnes i familie`() {
+        val behandlingId = UUID.randomUUID()
+        val grunnlagTestData = GrunnlagTestData()
+
+        coEvery {
+            grunnlagKlient.hentGrunnlag(any(), any())
+        } returns grunnlagTestData.hentOpplysningsgrunnlag()
+
+        repository.opprettTrygdetid(
+            trygdetid(
+                behandlingId = behandlingId,
+                sakId = SecureRandom().nextLong(100_000),
+                ident = "123",
+            ),
+        )
+
```

Comment:
greit å teste at denne fungerer med litt grunnlag / opplysninger lagt inn? hvis det ikke er for stress å sette opp testen